### PR TITLE
Ejether/minor fixes

### PR DESCRIPTION
**Pull request by ejether**
Some minor things I noticed and wanted to enhance.

### CODE REVIEW

**Comment by ejether** [comment]
I don't know a way around accepting the broken build. Since the passing the migration test requires the `--no-prompt` and this PR adds it. I guess, in order for these to pass, we'll have to remove the test, push a version with `--no-prompt`, release it, then add the migration test back in.

**Comment by endzyme** [comment]
@ejether you can add `yes | <pentagon commands>` which will stdout to stdin `y+Enter` for all command reads

Sorry edit: We should probably detect if the runenv is a tty or non-interactive. If non-interactive - don't ask acceptance. (https://rosettacode.org/wiki/Check_input_device_is_a_terminal#Python)

**Comment by ejether** [comment]
Ah! Why didn't I think of that. I'll give it a shot on Circle, then back out the `--prompt` stuff if it works.

File: cluster-config/cluster.yml [hunk 1]
```diff
@@ -0,0 +1,218 @@
+apiVersion: kops/v1alpha2
```
Comment (endzyme) [inline cluster-config/cluster.yml]: Did you mean to add all these files to the commit? This looks like the output from the pentagon migration run?

File: pentagon/component/__init__.py [hunk 2]
```diff
@@ -5,6 +5,7 @@
 import traceback
 import sys
 import re
+import click
```
Comment (endzyme) [inline pentagon/component/__init__.py]: You can remove this line

File: script.sh [hunk 3]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```
Comment (endzyme) [inline script.sh]: did you mean to commit this to the repo?

Comment (ejether) [reply script.sh]: Another good catch. I swear I was being careful

File: cluster-config/cluster.yml (hunk 1 above)
Comment (ejether) [reply cluster-config/cluster.yml]: Good catch, this should not be here.

**Review by endzyme — APPROVED** [review]

**Pull request merged**